File: CSHARP/CSHARP/Model/MjesanjeTriUnosa.cs
using System;
namespace CSHARP.Model
{
    public class MjesanjeTriUnosa:MjesanjeDvaUnosa
    {

        public int bodovaTreciUnos { get; set; }
        public int zvanjeTreciUnos { get; set; }


        public override Rezultat getRezultat()
        {
            var r = base.getRezultat();
            r.treci = bodovaTreciUnos + zvanjeTreciUnos;
            return r;
        }
    }
}

File: Kotlin/src/main/kotlin/model/MjesanjeTriUnosa.kt
package model

/**
 * @author Matija Skrletović
 **/

class MjesanjeTriUnosa(
    bodovaPrviUnos: Int,
    bodovaDrugiUnos: Int,
    zvanjePrviUnos: Int,
    zvanjeDrugiUnos: Int,
    val bodovaTreciUnos: Int,
    val zvanjeTreciUnos: Int = 0
) : MjesanjeDvaUnosa(
    bodovaPrviUnos, bodovaDrugiUnos, zvanjePrviUnos, zvanjeDrugiUnos
) {

    override fun getRezultat(): Rezultat {
        val treciRezultat = super.getRezultat()
        treciRezultat!!.treci = bodovaTreciUnos + zvanjeTreciUnos
        return treciRezultat
    }
}